thread/src/components/FooterTabs.js

```diff
@@ -17,15 +17,15 @@ const FooterTabs = ({ navigationState }) => {
     const { index, routes } = navigationState;
 
     return (
-        <Footer>
-            <FooterTab>
+        <Footer style={{height: 27}}>
+            <FooterTab style={{backgroundColor: '#66bb6a'}}>
```

Comment:
Please move the color to the global stylesheet in `native-base-theme/variables/platform.js`

Reply:
actually anywhere with colors, please place it there. If not put in a place where colors are not duplicated